src/values.js

```diff
@@ -343,10 +343,18 @@ Bytes.prototype.toJSON = function() {
 /** FaunaDB query. See the [docs](https://app.fauna.com/documentation/reference/queryapi#special-type).
  *
  * @param {any} value
+ * @param {any} api_version
  * @extends module:values~Value
  * @constructor
  */
 function Query(value) {
+  this.api_version = 'unstable'
```

Comment:
@marrony The ticket mentioned this as a possible value, but should `2.12` be the default?

Reply:
users should not be able to set this value, `Query` should be an opaque type